src/HelpText/widget/HelpTextAttr.js

```diff
@@ -29,7 +29,11 @@ define([
         text : "",
 
         _setValueAttr : function(value) {
-            this.text = value === null || typeof value === "undefined" ? "" : dom.escapeString(value).replace(/\n/g, "<br/>");
+            this.text = value;
+
+            if (!this.renderHTML) {
+                this.text = value === null || typeof value === "undefined" ? "" : dom.escapeString(value).replace(/\n/g, "<br/>");
+            }
```

Comment:
Although this is good, the function will fail when ``this.renderHTML === false`` and ``value === null``, because `dojo.html.set`` expects the second argument to be a string